README.md

```diff
@@ -1,18 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
-# Hello Microverse
+# Robin's Portfolio
 
 >
-This project shows how an html and css file works with tools like Linters to improve the developer's work performance
+This portfolio shows my achievements and what I have been able to develop thanks to my abilities by designing products in the programming world. If you have a new project and need code it, feel free to contact me.
 
+image.png
 
 ## Built With
 
-- HTML5/CSS.
+- HTML5.
+- CSS.
 - Visual Studio Code.
 
 
 ### Install
-Feel free to clone the repository at this URL: https://github.com/robinbenitezmora/Portfolio-Robin-Benitez/tree/Hello-Microverse
+Feel free to clone the repository at this URL: https://github.com/robinbenitezmora/Portfolio-Robin-Benitez
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: I think it would be better if you add some hints/processes about cloning the repository locally to make your documentation looks more professional. I suggest you take a look at **[this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md)** as an example. 

- [ **OPTIONAL** ] What do you think if you use the below format on the **Authors section** for your social links to make your README looks better and more professional. This will only show each **@username** which is clickable instead of the entire link.
```
## Authors

👤 ****Robin Benitez Mora****

- GitHub: [@robinbenitezmora](https://github.com/robinbenitezmora)
- LinkedIn: [Robin Benitez](https://www.linkedin.com/in/robin-benitez-mora-manizales/)
```

### NOTICE: Kindly make sure you also fix the requested change in the **review summary** about linter config. I put it there as I didn't find anywhere to write inline comments.